playwright/visual-test/InfoButton.spec.mjs

```diff
@@ -0,0 +1,8 @@
+// @flow strict
+import { test, expect } from '@playwright/test';
+
+test('Button visual regression check', async ({ page }) => {
+  await page.goto('/visual-test/InfoButton');
+  const locator = page.locator('[data-test-id="visual-test"]');
+  await expect(locator).toHaveScreenshot('InfoButton.png');
```

Comment:
I don't see this PNG in the PR?

Reply:
I need more resources about it, I don't know properly how it works and if we didn't have, why the tests is all greens?

Reply:
That's a good question. @AlbertCarreras Eduardo should need to add this PNG to the PR, yes?